robots_101/team_supervisor.md

```diff
@@ -7,17 +7,32 @@ title: Robots 101 - So you're running a team
 
 Here’s a little bit of info on what to expect from Student Robotics, and your responsibilities as a team supervisor.
 
-As a team supervisor, your role is to guide the competitors through the journey of building a robot. You’ll be there to point them in the right direction when they get stuck and resolve any issues they run into. We encourage team supervisors to limit their involvement with the robot design/building process so that the finished contraptions are 100% student-built which competitors find very rewarding.
+## Your role
+
+As a team supervisor, your role is to guide the competitors through the journey of building a robot. You’ll be there to point them in the right direction when they get stuck and resolve any issues they run into.
+You don't need to have any technical skills though -- our volunteer mentors can provide that [support](#support) either in person or remotely.
+We encourage team supervisors to limit their involvement with the [robot design]({{ site.baseurl }}/robots_101/design)/building process so that the finished contraptions are 100% student-built which competitors find very rewarding.
 
 You’ll be our point of contact with the team. If you have any questions during the year, just email <{{ site.emails.teams }}>. We’ll also be sending you emails every month or so with important information such as:
 - Details of [Tech Days]({{ site.baseurl }}/robots_101/tech_days) where we provide a space for teams to come together and get direct help from our dedicated volunteers
 - Software updates for our kit
-- Information about the competition event (usually in late March or April)
+- Information about the [competition event]({{ site.baseurl }}/robots_101/programme_structure#competition) (usually in late March or April)
 
 We aim to host our Kickstart event and Tech Days in multiple locations to make it more convenient for you to travel. However, you will still need to arrange to travel to these places.
 This is especially important to book for the competition, as you will likely need to arrange to stay overnight near the venue.
 
-Our kit includes a battery and a few boards to get your robot started. However, your team will need additional components and materials from which to build their robot. For the chassis, your team has several options. Teams often use cardboard, MDF, Aluminium, and/or Acrylic. In terms of electronic components, teams often make use of:
+## The team
+
+Teams usually consist of 4-8 competitors aged 16-19 and are often attending a school or college.
+However, the team doesn't necessarily need to represent a school -- any group of 16-19 year-olds, with a responsible adult, can enter.
```

Comment:
Is it worth defining what a `responsible adult` is here?

Reply:
I'd prefer not to -- I think it would be quite hard for one, and probably distract from what we're saying. I had however assumed that it would be a well understood phrase in the general context of things organised for children. Do you think that anyone reading this is likely to not understand what this means?

Reply:
Yeah, I think you're right. Looking at it again I can't really see a way someone who wants to supervise a team wouldn't know what that term meant...